src/Microsoft.Azure.ServiceBus/ServiceBusConnectionStringBuilder.cs

```diff
@@ -319,6 +332,17 @@ void ParseConnectionString(string connectionString)
                         this.TransportType = transportType;
                     }
                 }
+                else if (key.Equals(OperationTimeoutConfigName, StringComparison.OrdinalIgnoreCase))
+                {
+                    try
+                    {
+                        this.OperationTimeout = TimeSpan.Parse(value);
```

Comment:
Does everyone like using a TimeSpan format?
What does this mean?
OperationTimeout=60

What format do SQL connection strings use? (Seconds or something?)

Reply:
> What format do SQL connection strings use? (Seconds or something?)

SQL Server connection timeout is in seconds ([doco](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.connectiontimeout?view=netframework-4.7.2))

I'd vote to replace `TimeSpan.Parse()` with `TimeSpan.TryParse()` and then guard against values that are not reasonable/out of the acceptable range.

Reply:
I think it would be best to use seconds, just like SQL Server indeed. I like the idea to keep it as a `TimeSpan` instead of `int` since it’s self documenting.  I also like the idea to guard against values that are not reasonable. But how do we define this reasonable range? From 1 second to 1 hour? Any other suggestion?

Reply:
@nemakam what would be a reasonable value range in this case, can you suggest?

Reply:
@axisc could you please help to unblock this PR? It's almost done and just requires some feedback from the team. Thank you.

Reply:
Its a fair question. Do we keep the `ToString()` in the `TimeSpan` format, or numberOfSeconds format. Keeping in `TimeSpan` format makes it interchangable with older client, and keeping it numberOfSeconds will make it easier across clients (not just .net and java, but also the other languages like python, node, go etc etc). 
Let me discuss with the broader team and see what they think about this...

Reply:
So here's the current status -
Old .net client uses Timespan.Parse() and Timespan.ToString() , where format is 00:00:60.
Java client uses Duration, which has its own format.
Moving forward, `seconds` feel like a safe bet as the format for operation timeout. So how about this - 

while parsing, we do
```csharp
if (int.TryParse(operationTimeoutInputValue, out var ot))
{
    operationTimeout = TimeSpan.FromSeconds(ot);
}
else
{
    TimeSpan.TryParse(operationTimeoutInputValue, out operationTimeout);
}
```

While doing ToString(), use `operationTimeout.TotalSeconds`.

The outcome is that in new world we will end up using seconds to keep it less confusing about how to parse, but we also let users migrate from old .net client to new .net client.